Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -179,6 +193,12 @@ internal class DebugView: UIView {
     }
 
     override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+
+        // any interaction while toast is showing will initiate dismiss
+        if toastWrapperView.alpha > 0 {
+            animateToast(visible: false, animated: true, completion: nil)
+        }
```

Comment:
This is very clever! Almost feels too clever but it'd be a bunch more code to do a different way.